### PR TITLE
add prazeres violentos

### LINKED ISSUE #3032: Prazeres Violentos
### Source name

Prazeres Violentos

### Source link

https://prazeresviolentos.com/

### Source language

Portuguese 

### Other details

Password: casadeprazeres

### Acknowledgements

- [X] I have checked that the extension does not already exist by searching the [extension listing](https://keiyoushi.github.io/extensions/) and verified it does not appear there.
- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open or closed issue.
- [X] I have written a meaningful title with the source name.
- [X] I will fill out all of the requested information in this form.

### PR DESCRIPTION
**Pull request by kana-shii**
closes #3032
Checklist:

- [ ] Updated `extVersionCode` value in `build.gradle` for individual extensions
- [ ] Updated `overrideVersionCode` or `baseVersionCode` as needed for all multisrc extensions
- [x] Referenced all related issues in the PR body (e.g. "Closes #xyz")
- [x] Added the `isNsfw = true` flag in `build.gradle` when appropriate
- [x] Have not changed source names
- [ ] Have explicitly kept the `id` if a source's name or language were changed
- [x] Have tested the modifications by compiling and running the extension through Android Studio
- [x] Have removed `web_hi_res_512.png` when adding a new extension

### CODE REVIEW

File: src/pt/prazeresviolentos/src/eu/kanade/tachiyomi/extension/pt/prazeresviolentos/PrazeresViolentos.kt
Comment (AwkwardPeak7) [inline src/pt/prazeresviolentos/src/eu/kanade/tachiyomi/extension/pt/prazeresviolentos/PrazeresViolentos.kt:1]: the site requires a password to access, you can either throw appropriate exception message (telling user to login via webview) when password protection is detected or add mechanism to login automatically using password from preference screen

Comment (kana-shii) [reply src/pt/prazeresviolentos/src/eu/kanade/tachiyomi/extension/pt/prazeresviolentos/PrazeresViolentos.kt:1]: Is there any already built extension that uses the second option? I've been curious about adding that, but I don't know how.

Comment (AwkwardPeak7) [reply src/pt/prazeresviolentos/src/eu/kanade/tachiyomi/extension/pt/prazeresviolentos/PrazeresViolentos.kt:1]: not sure if there are any existing examples, but you can detect login page in interceptor and attempt login there using credentials from sharedreference. if credentials are wrong or missing, throw exception asking user to enter new credentials.

**Comment by AwkwardPeak7** [comment]
build failed btw

**Review by AwkwardPeak7 — APPROVED** [review]

**Pull request merged**